contrib/perf-check-status.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
this script might be more suitable for the cli itself. having a `edgectl health` or similar would be great